### ISSUE TITLE
Documentation update for thick element tracking 

### ISSUE THREAD

**Issue by chernals** [MethodicalAcceleratorDesign/MAD-X#849]
It seems that a few items are outdated.

For example, following #113 and based on a few tests, it seems that thick tracking in quadrupoles is now the default.

The following, in `elements.tex` could/should be changed.

```
   \ttitem{THICK} If this logical flag is set to true the quadrupole
   is tracked through as a thick-element, instead of being
   converted into thin-lenses.\\
   (Default: false)
```

Same goes for `SBEND` or `RBEND`:

```
   \ttitem{THICK} If this logical flag is set to true the bending
   magnet is tracked through as a thick-element, instead of being
   converted into thin-lenses. \\
   (Default: false)
```

Setting it one way or another does not seem to have any effect.

**Comment by tpersson** [comment]
The explanation in the documentation is not very clear. The thick attribute has an impact on the makethin command. If present then it will still be a thick element after the command and otherwise a multipole.

**Comment by tpersson** [comment]
Fixed in #851
